AzureCommunicationUIDemoApp/App.js

```diff
@@ -265,7 +268,7 @@ const App = () => {
               </View>
               <View style={styles.settingsSectionContainerView}>
                 <Text style={styles.settingsHeaderText}>Localization</Text>
-                <RNPickerSelect
+                {/* <RNPickerSelect
```

Comment:
is this package still causing issue? can we try to update and see if it works?

Reply:
Yes. I tried with different version. The picker is making build error due to underlaying version missmatch. I also removed it from the package dependency. 
We can update UI with stable components to select assets when we take on the UI upgrade work.

Reply:
@iaulakh With upgrade to 0.72 of React-Native, the issue for picker has been resolved. Please check to validate for the release.